source/core/Math.cpp

```diff
@@ -278,6 +276,16 @@ using namespace std;
 		return _NextInstruction();													\
 		}
 
+#define DECLARE_VIREO_COMPARISON_PRIMITIVES(TYPE) \
+    DECLARE_VIREO_PRIMITIVE2( IsLT0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) <  0) ) \
+    DECLARE_VIREO_PRIMITIVE2( IsGE0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) >= 0) ) \
```

Comment:
No, the IsLE0 and IsGT0 don't generate warnings.  The warnings are because Boolean can only be 0 or 1, and saying "if (b < 0)" when b can only be 0 or 1, or "if (b >= 0)", doesn't make sense.  
However, if "(b <= 0)" does make sense, and is sometimes true, sometimes false.